Packs/RecordedFutureASI/Integrations/RecordedFutureASI/RecordedFutureASI.py

```diff
@@ -532,8 +531,12 @@ def fetch_incidents(client: Client, last_run: Dict[str, int], is_by_host: bool,
 
 def main() -> None:
     params = demisto.params()
-    api_key = params.get('apikey')
-    project_id = params.get('project_id')
+    api_key = params.get("credential_apikey", {}).get("password") or params.get("apikey")
+    if not api_key:
+        return_error('Please provide a valid API token')
+    project_id = params.get("credential_projectid", {}).get("password") or params.get("project_id")
+    if not project_id:
+        return_error('Please provide a valid Project ID')
```

Comment:
Please make the credentials entry one entry in the credentials vault where the project_id is the username  and the apikey is the password. While still allowing the old way of authenticating where both project_id and apikey is a secret. Reach out in slack @NicCas if you have any questions